CSC8503/InventoryBuffSystem/PlayerInventory.cpp

```diff
@@ -103,6 +103,10 @@ bool InventoryBuffSystem::PlayerInventory::HandleOnItemUsed(item item, int playe
 	return false;
 }
 
+ItemUseType InventoryBuffSystem::PlayerInventory::GetItemUseType(item item) {
```

Comment:
Using distinct names for the type and parameter would be better for clarity.

Reply:
Done 👍